anonymizer/OpenDiffix.Service.Tests/Program.Tests.fs

```diff
@@ -51,17 +54,46 @@ let ``Handles Preview request`` () =
   response |> should haveSubstring "medianDistortion"
   response |> should haveSubstring "rows"
 
+[<Fact>]
+let ``Handles Preview request with custom anonParams`` () =
+  let requestCustom =
+    $"""
+    {{"type":"Preview","inputPath":"%s{dataPath}","aidColumn":"id","salt":"1","anonParams":{{"suppression":{{"lowThreshold":0,"sD":0,"lowMeanGap":0}},"outlierCount":{{"lower":2,"upper":5}},"topCount":{{"lower":2,"upper":5}},"noiseSD":1.0}},"buckets":["age","city"],"countInput":"Rows","rows":1000}}
```

Comment:
These lines are pretty long and hard to read. Can you please split them into multiple lines?

Reply:
Also why not send an object to the test? We can assume (or test somewhere else) that JSON parsing works as intended.

Reply:
I've split the longest ones somewhat, following the structure of the request payload.

For now this is a simple smoke test to ensure the stack works, in particular, that the requests from `anonymizer.ts` work. It's very convenient to copy over the request string from logs into the test. I'm not sure we're investing time into more fine-grained testing at this point.